src/ui/buttons/button.jsx

```diff
@@ -263,7 +263,7 @@ export function Button({ fundingSource, style, multiple, locale, env, fundingEli
                 <Spinner />
             </div>
 
-            { shouldShowWalletMenu ? <MenuButton textColor={ textColor } /> : null }
+            { shouldShowWalletMenu ? <MenuButton textColor={ textColor } content={ content } /> : null }
```

Comment:
will content ever be used for anything other than `aria-label`? I was thinking maybe:

```
<MenuButton aria-label={content?.moreOptions} ... />
```

Reply:
Thanks for the feedback, @jshawl! At the moment, `content` isn't used for anything in `MenuButton` except the aria-label, but that might not always be the case. I'm not seeing any existing patterns where individual `content` properties are passed down to child components, but instead, I do see a pattern of passing down the entire `content` object. I'm leaning towards keeping it as is, unless you can think of any other reason for only passing down `content.moreOptions` to `MenuButton`.